src/fields/ed25519Fr.nr

```diff
@@ -1,16 +1,19 @@
+use crate::bignum::BigNum;
 use crate::params::BigNumParams;
 use crate::params::BigNumParamsGetter;
 use crate::utils::u60_representation::U60Repr;
 
 pub struct ED25519_Fr_Params {}
 
-impl BigNumParamsGetter<3, 255> for ED25519_Fr_Params {
-    fn get_params() -> BigNumParams<3, 255> {
+pub type ED25519_Fr = BigNum<3, 253, ED25519_Fr_Params>;
+
+impl BigNumParamsGetter<3, 253> for ED25519_Fr_Params {
```

Comment:
Error.

Reply:
cc @kashbrti any idea why the paramgen tool might be broken here? I generated from your PR (although of course the bug might have originated earlier)

Reply:
I've manually fixed it, for now, so that this PR can be merged.